Jenkinsfile

```diff
@@ -33,9 +33,7 @@ pipeline {
           if (isFullRelease(TAG)) {
             docker.withRegistry('https://index.docker.io/v1/', '8a04e3ab-c6db-44af-8198-1beb391c98d2') {
               def image = docker.build("instana/instana-agent-operator:$VERSION", BUILD_ARGS)
-
               image.push()
-              image.push('latest')
```

Comment:
@wiggzz After discussion with Henning/Miel/Et al., we decided to no longer push the `latest` tag and to make the dependency between the operator image and the operator yaml more explicit, i.e. if a customer wants a more recent version of the operator, they have to update their operator yaml.